<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertsDialog</class>
 <widget class="QDialog" name="AlertsDialog">
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Справочник событий</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QLabel" name="dateFromLabel">
       <property name="text">
        <string>С:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateFromEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dateToLabel">
       <property name="text">
        <string>По:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateToEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Категория:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryCombo">
       <property name="minimumWidth">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Экспорт в CSV</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="columnCount">
      <number>8</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Категория</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Событие</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пользователь</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сотрудник</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сообщение</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
